Deploy-SPVariation.md

```diff
@@ -0,0 +1,135 @@
+---
+external help file: Microsoft.SharePoint.Publishing.dll-help.xml
+Module Name: SharePointServer
+online version:
+schema: 2.0.0
+---
+
+# Deploy-SPVariation
+
+## SYNOPSIS
+Creates new site variations on the source variation site specified by the Identity parameter for all target variations labels.
+
+## SYNTAX
+
+```
+Deploy-SPVariation -Identity <SPWebPipeBind> [-Recurse] [-Label <String>]
+ [-AssignmentCollection <SPAssignmentCollection>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The command creates new site variations on the source variation site specified by the Identity parameter for all target variations labels.
+If the Recurse parameter is used, variations for subsites and pages are also created.
+
+## EXAMPLES
+
+### EXAMPLE
+```
+C:\PS> Deploy-SPVariation -Identity https://server/sites/pub/vhome/source/newsub
+```
+
+The example is to create new site variations on the source variation site specified by the Identity parameter for all target variations labels.

```

Comment:
```suggestion
The example creates new site variations on the source variation site specified by the Identity parameter for all target variations labels.

```

Reply:
Thanks for your suggestions. Have updated.